Patch-Source: https://github.com/hrydgard/ppsspp/commit/23ca9f629424a9c5e48fb2c58488fda871e1017f
From 6a9334e6f01b0c9b8a31f791f968851d4141d7a1 Mon Sep 17 00:00:00 2001
From: Jan Beich <jbeich@FreeBSD.org>
Date: Thu, 15 Dec 2022 15:31:04 +0000
Subject: [PATCH] OpenXR - Correct NULL type for XrSession

Common/VR/VRBase.cpp:220:30: error: assigning to 'XrSession' (aka 'unsigned long long') from incompatible type 'nullptr_t'
                engine->appState.Session = NULL;
                                           ^~~~
/usr/include/sys/_null.h:37:14: note: expanded from macro 'NULL'
 #define NULL    nullptr
                 ^~~~~~~
---
 Common/VR/VRBase.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Common/VR/VRBase.cpp b/Common/VR/VRBase.cpp
index 2cb2d0557dd..edc9cc84dae 100644
--- a/Common/VR/VRBase.cpp
+++ b/Common/VR/VRBase.cpp
@@ -217,7 +217,7 @@ void VR_LeaveVR( engine_t* engine ) {
 		OXR(xrDestroySpace(engine->appState.FakeStageSpace));
 		engine->appState.CurrentSpace = XR_NULL_HANDLE;
 		OXR(xrDestroySession(engine->appState.Session));
-		engine->appState.Session = NULL;
+		engine->appState.Session = XR_NULL_HANDLE;
 	}
 }
 
